topology/University of Colorado - Boulder/Research Computing/CUmulus.yaml

```diff
@@ -0,0 +1,1366 @@
+      Administrative Contact:
+        Primary:
+          Name: Brett Shouse
+          ID: 25da6f6a72dfe6fc8b25da208a7c69d81171f29e
+        # Secondary:
+        #   Name: <FIRSTNAME> <LASTNAME>
+        #   ID: <EMAIL HASH>
+        # Tertiary:
+        #   Name: <FIRSTNAME> <LASTNAME>
+        #   ID: <EMAIL HASH>
+
+      # Security Contact is one to three people to contact regarding security issues
+      Security Contact:
+        Primary:
+          Name: Brett Shouse
+          ID: 25da6f6a72dfe6fc8b25da208a7c69d81171f29e
+        # Secondary:
+        #   Name: <FIRSTNAME> <LASTNAME>
+        #   ID: <EMAIL HASH>
+        # Tertiary:
+        #   Name: <FIRSTNAME> <LASTNAME>
+        #   ID: <EMAIL HASH>
+
+    # FQDN is the fully qualified domain name of the host running this resource
+    FQDN: porter3s5.rc.int.colorado.edu
+    ### FQDNAliases (optional) are any other DNS aliases by which this host can be accessed
+    # FQDNAliases:
+    #   - <HOSTNAME1>
+    #   - <HOSTNAME2>
+
+    ### DN (optional except for XCache resources) is the DN of the host cert of the resource
+    # in OpenSSL 1.0 format (i.e. /DC=org/DC=incommon/C=US/...)
+    # DN: <DN>
+
+    # Services is one or more services provided by this resource;
+    # valid services are listed in topology/services.yaml with the format "<SERVICE NAME>: <ID>"
+    Services:
+      Virtual Machine Host:
+        # Description is a brief description of the service
+        Description: a compute node hosting virtual machines
+        ### Details (optional)
+        # Details:
+        #   # hidden
+        #   hidden: false
+        #   ### uri_override (optional, use if your service is on some non-standard URL)
+        #   # uri_override: <HOST>:<PORT>
+        #   ### sam_uri (optional)
+        #   # sam_uri: htcondor://...
+        #   ### endpoint (for perfSONAR services)
+        #   # endpoint: <HOSTNAME>
+
+      # Other services if you have any
+      # <SERVICE NAME>:
+      # ...
+
+    ### Tags (optional) is a list of tags associated with the resource.
+    ### Include the tag "CC*" if applicable for a CC* CE.
+    # Tags:
+    #   - <TAG1>
+    #   - <TAG2>
+
+    ### VOOwnership (optional) is the percentage of the resource owned by one or more VOs.
+    ### If part of the resource is not owned by the VO, do not list it.
+    ### The total percentage cannot exceed 100.
+    # VOOwnership:
+    #   <VO1>: <PERCENT>
+    #   <VO2>: <PERCENT>
+
+    ### WLCGInformation (optional) is only for resources that are part of the WLCG
+    # WLCGInformation:
+    #   APELNormalFactor: 0.0
+    #   AccountingName: <name>
+    #   HEPSPEC: 0
+    #   InteropAccounting: true
+    #   InteropBDII: true
+    #   InteropMonitoring: true
+    #   KSI2KMax: 0
+    #   KSI2KMin: 0
+    #   StorageCapacityMax: 0
+    #   StorageCapacityMin: 0
+    #   TapeCapacity: 0
+
+  # Other resources if you have any...
+  # <RESOURCE NAME>:
+  # ...
```

Comment:
```suggestion
```

Confusingly, we don't need to register each of the VMs in question. Just a single "endpoint" is fine for our purposes!